src/bridge/cockpitdbusjson.c

```diff
@@ -1998,6 +2000,12 @@ cockpit_dbus_json_prepare (CockpitChannel *channel)
       goto out;
     }
 
+  if (!internal && !cockpit_dbus_json_allow_external)
+    {
+      problem = "not-supported";
```

Comment:
Since this is security related, we should add a test to make sure it doesn't regress.